src/librawspeed/decoders/DngDecoderSlices.cpp

```diff
@@ -418,10 +419,9 @@ void DngDecoderSlices::decodeSlice(DngDecoderThread* t) {
     while (!t->slices.empty()) {
       DngSliceElement e = t->slices.front();
       t->slices.pop();
-      LJpegPlain l(*mFile, e.byteOffset, e.byteCount, mRaw);
-      l.mDNGCompatible = mFixLjpeg;
+      DngDecompressor d(*mFile, e.byteOffset, e.byteCount, mRaw);
```

Comment:
> since the last commit already moves stuff around, it kind-of makes sense here.

Except I did not move/rename _this_ particular class. So it is not like it would need to be renamed again. So merging it as is (maybe rename the DngDecompressor before - see above) and then doing this other improvement later, makes even more sense.